replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+---
+title: "replication"
+author: "Nick F & Aaron K"
+date: "March 15, 2015"
+output: html_document
+---
+
+```{r echo = FALSE}
+suppressPackageStartupMessages(require(dplyr))
+suppressPackageStartupMessages(require(tidyr))
+suppressPackageStartupMessages(require(aod)) # wald.test
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr)) #etaSquared
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(foreign))
+
+# detect the user's directory:
+directory  <- "C:/Users/Nick/Documents/Github/scarcityvalueRep"
+if (Sys.getenv('USER')=="air") directory <- "~/Dropbox/SOC504/replicate"
+setwd(directory)
+
```

Comment:
OK, but this system would not work if someone tried to download your code and run it on their machine.  Ideally the code would be totally portable to any machine and any operating system.  Something to consider for the next project . . .